src/app/component/ImageUpload/index.js

```diff
@@ -0,0 +1,82 @@
+import React, { useRef, useState } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { ReactComponent as CameraIcon } from '../../../img/camera.svg';
+import ImagePreview from './ImagePreview';
+import Button from '../Button/Button';
+
+export function ImageUpload({ styles, getFile = () => null }) {
+  const fileUpload = useRef();
+  const [file, setFile] = useState('');
+  const [preview, setPreview] = useState(null);
+
+  function handleFileChange({ target }) {
+    setFile(target.files[0]);
+    getFile(target.files[0]);
+    target.files.length && setPreview(URL.createObjectURL(target.files[0])); // Sanity check to only call createObjectURL when there is file
+  }
+
+  function removeImage() {
+    setFile(null);
+    getFile(null);
+    setPreview(null);
+  }
+
+  function onTakeImage() {
+    fileUpload.current.click();
+  }
+
+  return (
+    <Container style={styles}>
+      <StyledIcon>
+        {file ? (
+          <ImagePreview removeImage={removeImage} preview={preview} file={file} />
+        ) : (
+          <CameraIcon data-testid="cameraIcon" />
+        )}
+      </StyledIcon>
+      <Actions>
+        <Button onClick={onTakeImage} text="Take Photo" />
+        <Button tertiary onClick={onTakeImage} text="Upload Photo" />
+      </Actions>
+      <input
+        onChange={handleFileChange}
+        onClick={onTakeImage}
+        data-testid="uploadField"
+        ref={fileUpload}
+        type="file"
+        accept="image/x-png,image/gif,image/jpeg capture=camera"
+        hidden
+      />
+    </Container>
+  );
+}
+
+const Container = styled.div`
```

Comment:
Based on quick chat from @mat10tng, we might want to move the styled components into ImageUpload.style.js, if I understood him correctly

Reply:
yes, I have had some discussion with Terrence.